no-computer.md

```diff
@@ -11,41 +11,106 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A group of people on campus regularly play a large game of Diplomacy.
+The board games involve the players sending in instructions for their units
+on the board.
+These instructions are collected then played out simultaneously.
+This causes a lot of conflicts between their instructions which are then 
+resolved based on the game's rules.
+Further complications arise when players send invalid instructions.
+
+A program that can take in these instructions and then validate moves and
+resolve conflicts would speed up this process.
+As of now, it takes 5 to 10 minutes for people to manage this process.
 
```

Comment:
Wow that is a long time to validate the moves/conflicts. A DSL could be really helpful with speeding up the process. I naturally? question if this will be useful outside of Mudd.